kubernetes/samples/scripts/create-rcu-schema/start-db-service.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+
+# Bring up Oracle DB Instance in [default] NameSpace with a NodePort Service 
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+
+function usage {
+  echo "usage: ${script} -p <nodeport> [-h]"
+  echo "  -p DBService External NodePort"
+  echo "  -h Help"
+  exit $1
+}
+
+while getopts "h:p:" opt; do
+  case $opt in
+    p) nodeport="${OPTARG}"
+    ;;
+    h) usage 0
+    ;;
+    *) usage 1
+    ;;
+  esac
+done
+
+if [ -z ${nodeport} ]; then
+  nodeport=30011
+fi
+
+
+ocr_pfx=container-registry.oracle.com
+db_image=${ocr_pfx}/database/enterprise:12.2.0.1-slim
+docker pull ${db_image}
```

Comment:
authentication?

Reply:
Now I  check the result for docker pull command and provide the suggested action as follows e.g. in start-db-service.sh

ocr_pfx=container-registry.oracle.com
db_image=${ocr_pfx}/database/enterprise:12.2.0.1-slim
docker pull ${db_image}
if [ $? != 0 ]; then
echo "[ERROR] Could not pull ${db_image}";
echo "Please run [ docker login ${ocr_pfx} ] and "
echo "Check-out the db image (if needed)"
exit -1;
fi